Orabot.Core/Modules/OpenRaWeaponsModule.cs

```diff
@@ -9,51 +8,68 @@ namespace Orabot.Core.Modules
 {
 	public class OpenRaWeaponsModule : ModuleBase<SocketCommandContext>
 	{
-		private const string WeaponsPageUrl = "https://docs.openra.net/en/release/weapons/";
-		private const string WeaponsPlaytestPageUrl = "https://docs.openra.net/en/playtest/weapons/";
-
 		private readonly IRestClient _restClient;
 		private readonly string _openRaIconUrl;
+		private readonly string _weaponsReleasePageUrl;
+		private readonly string _weaponsPlaytestPageUrl;
+		private readonly string _weaponsDevelopmentPageUrl;
 
 		public OpenRaWeaponsModule(IRestClient restClient, IConfiguration configuration)
 		{
 			_restClient = restClient;
-			_restClient.BaseUrl = new Uri(WeaponsPageUrl);
+
 			_openRaIconUrl = configuration["OpenRaFaviconUrl"];
+
+			var traitsPages = configuration.GetRequiredSection("Weapons");
+			_weaponsReleasePageUrl = traitsPages["ReleasePageUrl"];
+			_weaponsPlaytestPageUrl = traitsPages["PlaytestPageUrl"];
+			_weaponsDevelopmentPageUrl = traitsPages["DevelopmentPageUrl"];
 		}
 
 		[Command("weapons")]
 		[Summary("Provides a link to the OpenRA Weapons documentation page. Can be used with an optional weapon name to link directly.")]
 		public async Task Weapons(string weaponName = null)
 		{
-			var embed = BuildWeaponsPageEmbed(WeaponsPageUrl, weaponName);
-			await ReplyAsync("", false, embed);
+			var embed = await BuildWeaponsPageEmbed(_weaponsReleasePageUrl, weaponName);
+			if (embed != null)
+				await ReplyAsync("", false, embed);
 		}
 
 		[Command("weapons-pt")]
 		[Summary("Provides a link to the OpenRA playtest Weapons documentation page. Can be used with an optional weapon name to link directly.")]
 		public async Task WeaponsPt(string weaponName = null)
 		{
-			var embed = BuildWeaponsPageEmbed(WeaponsPlaytestPageUrl, weaponName);
-			await ReplyAsync("", false, embed);
+			var embed = await BuildWeaponsPageEmbed(_weaponsPlaytestPageUrl, weaponName);
+			if (embed != null)
+				await ReplyAsync("", false, embed);
+		}
+
+		[Command("weapons-dev")]
+		[Summary("Provides a link to the OpenRA playtest Weapons documentation page. Can be used with an optional weapon name to link directly.")]
```

Comment:
```suggestion
		[Summary("Provides a link to the OpenRA development Weapons documentation page. Can be used with an optional weapon name to link directly.")]
```

Reply:
Looks like I pushed the fix for that before you commented :P